Mockups/Base/MockCubeBlock.debug.cs

```diff
@@ -71,12 +62,12 @@ public override string ToString()
         {
             var name = GetType().Name + " #" + EntityId;
 
-            if (!string.IsNullOrWhiteSpace(DisplayNameText))
+            if (!string.IsNullOrWhiteSpace(DisplayName))
             {
-                name += ": " + DisplayNameText;
+                name += ": " + DisplayName;
```

Comment:
Why was this done?

Reply:
`DisplayNameText` wasn't set during mock runs, whereas `DisplayName` was.